### PR TITLE
use fake data on dev

### PR DESCRIPTION
**Pull request by farisashai**
## Details 
If DEV_MODE flag is set to true, we wont fetch real data to save API points for testing purposes

## New Changes
- able to test project without actual data

## Checklist
 If it doesn't apply, just mark as completed
- [x] I have requested 2+ code reviewers
- [x] I have reviewed and commented my own code.
- [x] My code follows the style guidelines of this project.
- [x] I have added new tests as necessary.
- [x] All new and existing tests pass successfully.

### CODE REVIEW

**Review by hanran-y — APPROVED** [review]
wow, this looks real.

**Review by treeebooor — APPROVED** [review]
Woohoo very nice!! But then we still have to use real data if we want to test searching or filtering. But yeah woohoo!!

File: source/scripts/constants.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const API_KEY = '857b051abfmsh018f07c4f390b41p1547b1jsn64ae2598745b';// prevent exposing api key
```
Comment (RainbowPangolin) [inline source/scripts/constants.js:1]: Perhaps renaming this to config.js would make it a little more obvious what it means.

**Review by RainbowPangolin — COMMENTED** [review]
Consider renaming constant.js to config.js, but other than that it seems fine.

So when testing, we'd have to change the flag in the js file? I tried doing that on the Netlify preview but it didn't allow the change- so this would only really apply on local testing right?

**Pull request merged**